action.yaml

```diff
@@ -31,8 +31,10 @@ runs:
     # This is to avoid RabbitMQ to fail at startup because a wrong version of erlang was installed
     - name: Workaround for RabbitMQ
       run: |
+        # fix freaky leading "-e" in /etc/hosts
```

Comment:
I know this is merged now, but it would be good to link to the issue so that we know if/when we can remove this later.

Reply:
done in main